test/one/profiler/test/TestProcess.java

```diff
@@ -279,6 +279,10 @@ public int exitCode() {
         return p.exitValue();
     }
 
+    public boolean processIsAlive() {
```

Comment:
The class is already named `TestProcess`, another `process` in the name is redundant.